addons/hr_expense/models/hr_expense.py

```diff
@@ -313,6 +313,7 @@ def _get_account_move_line_values(self):
                 'expense_id': expense.id,
                 'partner_id': partner_id,
                 'tax_ids': [(6, 0, expense.tax_ids.ids)],
+                'tag_ids': [(6, 0, taxes['base_tags'])],
```

Comment:
that's weird, the 6 command removes all before adding. Shouldn't it be `'tag_ids': [(6, 0, expense.tax_ids.ids - taxes['base_tags'])],`?

Reply:
Nope, we're setting tags here, not taxes. 'taxes' variable is returned by the call to compute_all. The 'base_tags' field contains all the ids of the tags to be applied to the base line, i.e. the one with the tax_ids field set.

Reply:
how yeah, indeed, sorry

Reply:
@nim-odoo Un p'tit r+ alors ? :)